Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -86,7 +87,7 @@ int ApplicationMainStart(const char* principalName,
         } else if ([orientation isKindOfClass:[NSArray class]]) {
             bool found = false;
 
-            for (NSString* curstr in (NSArray*)orientation) {
+            for (NSString* curstr in(NSArray*)orientation) {
```

Comment:
The pre-commit hook will not let me keep the space, looking into it

---
In reply to: [70532438](https://github.com/Microsoft/WinObjC/pull/637#discussion_r70532438) [](ancestors = 70532438)